synapse/api/room_versions.py

```diff
@@ -244,24 +244,8 @@ class RoomVersions:
         msc2716_historical=False,
         msc2716_redactions=False,
     )
-    MSC2716 = RoomVersion(
```

Comment:
Dropping support for older MSC2716 room versions so we don't have to worry about
supporting both chunk and batch events.